page-objects/pages/TASKListPage.js

```diff
@@ -159,60 +159,58 @@ async function clickDeleteTaskButtonInPopup() {
 }
 
 async function clickAtTask(taskName) {
```

Comment:
I think we need to invest more time into this. Cleaning up clickOnTask, clickOnTaskFromList, clickAtTask and so on. There's to much stuff which just leads to confusion and we don't need functions that do similar stuff.
We need one method for one action/object. For example one method that will get the list of tasks. Closed. Next method to check if specific task with name is there. Closed. Then another method to check something regarding that task. And so on.

Maybe as a first step, we should write down, what we actually need to do and then will see which steps are redundant and dependent of each other. Then we should do a review of what have and refactor/clean up.

Reply:
OK, I'll add my comment from here to the ticket.